Dockerfile

```diff
@@ -2,11 +2,17 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR /code
 
 # Expose port 8000 for your FastAPI server.
+EXPOSE 8000
 
 # Install Python requirements using Pip.
+COPY requirements.txt ./code/
```

Comment:
You're copying your requirements to the wrong location.
(Hint: In the `pip install` command, you're referring to the right location)